oak-doc/src/site/markdown/query/lucene.md

```diff
@@ -845,7 +850,17 @@ all the other components (e.g. `charFilters`, `Synonym`) are optional.
 
 #### Examples
 
-Adding stemming support
+To convert umlauts using ASCII folding, use:
```

Comment:
Converting umlauts is just an example of what ASCII folding does, but the sentence above sounds like the example below is just for umlauts. Maybe rephrase to explain that umlaut conversion is just an example and show what it does: Ü -> u. And give another example, for instance: `açaí` -> `acai`. (taken from the Elasticsearch documentation, https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-asciifolding-tokenfilter.html)

Reply:
I was more coming from the use case... The use case is "umlauts", and one of the solutions for that (probably the easiest) is to use ASCII folding. What you suggest is coming from the ASCII folding, and then list what can you do with it. Sure, we can do that as well; however, to avoid copyright problems, we can't just copy & paste Elasticsearch docs.

Reply:
I was not suggesting to copy'n'paste, just use as an example the word `açaí`. Otherwise, I can suggest copywrite-free Portuguese words: Conceição, aço, cão, coração... :) But I think most people will know açaí better. But your changes are good, now it's more clear.